libpod/volume_internal_linux.go

```diff
@@ -33,7 +33,7 @@ func (v *Volume) mount() error {
 	}
 
 	// We cannot mount 'local' volumes as rootless.
-	if !v.UsesVolumeDriver() && rootless.IsRootless() {
+	if rootless.IsRootless() && v.RequiresRoot() {
```

Comment:
You need to retain the `!UsesVolumeDriver()` - that catches volume plugins

Reply:
@mheon what drivers are not supported in rootless mode?
I think "local", "bind", "tmpfs" and "" are all supported.  If all UserVolumeMounts are supported, or at least allow them to fail themselves.  I

Reply:
Bind and tmpfs are not drivers, they're methods the local plugin can use to mount.

The local driver (and "", an alias to it) is supported rootless, in a limited fashion - only the kernel mount types that can be used in a rootless user namespace will work.

Any drivers that aren't local are supported (these would be volume plugins), though we make no promises that a given volume plugin actually works with rootless Podman.

Reply:
I removed both checks, just allow the mount and unmount commands to fail.  Don't try to decide what is supported and what is not.

Reply:
Works for me